### PR TITLE
Updated names and labels

### PR DESCRIPTION
**Pull request by stepps00**
This PR attempts to update name translations for top-level placetypes (country, macroregion, region, county) and large localities (based on population) for records in this repo.

In this PR, English names, labels, and `wof:name` values are being updated. I have additional, localized name translations for records in other admin repos.. if these changes look good, I'll apply the same work to a handful of other repos for QA, then merge all directly to the main branch.

### CODE REVIEW

File: data/856/706/83/85670683.geojson [hunk 1]
```diff
@@ -34,8 +34,10 @@
         "Ain Temouchent"
     ],
     "name:ara_x_preferred":[
-        "'A\u00efn Temouchent",
-        "\u0648\u0644\u0627\u064a\u0629 \u0639\u064a\u0646 \u062a\u0645\u0648\u0634\u0646\u062a"
+        "\u0639\u064a\u0646 \u062a\u0645\u0648\u0634\u0646\u062a"
```
Comment (nvkelso) [inline data/856/706/83/85670683.geojson:37]: Uff, this is a good fix.

File: data/856/706/99/85670699.geojson [hunk 2]
```diff
@@ -30,8 +30,10 @@
     "mz:max_zoom":11.0,
     "mz:min_zoom":7.7,
     "name:ara_x_preferred":[
-        "Bechar",
-        "\u0648\u0644\u0627\u064a\u0629 \u0628\u0634\u0627\u0631"
+        "\u0628\u0634\u0627\u0631"
+    ],
+    "name:ara_x_variant":[
```
Comment (nvkelso) [inline data/856/706/99/85670699.geojson:35]: This is a latin script variant, do we have special property name for that?

**Review by nvkelso — APPROVED** [review]
Some nits and exclamations, but LGTM.

File: data/856/706/99/85670699.geojson (hunk 2 above)
Comment (stepps00) [reply data/856/706/99/85670699.geojson]: Resolved in 9c20bbd.. I edited this name, storing it in a `name:ara_latn_x_preferred` property.

**Pull request merged**